python-manual/modules/ROOT/pages/transformers.adoc

```diff
@@ -0,0 +1,225 @@
+= Manipulate query results
+
+This section shows how to work with a query's result so as to extract data in the form that is most convenient for your application.
+
+
+== Result as a list
+
+By default, xref:query-simple.adoc[`Driver.execute_query()`] returns an link:{neo4j-docs-base-uri}/api/python-driver/current/api.html#neo4j.EagerResult[`EagerResult`] object.
+It contains the result records as a list, so it is easy to loop through them *(1)*.
+It also contains a link:{neo4j-docs-base-uri}/api/python-driver/current/api.html#resultsummary[`ResultSummary`] of the query result, with metadata and information about the result *(2)*, and the `keys` available in the returned rows *(3)*.
+
+[source, python]
+----
+records, summary, keys = driver.execute_query(
+    "MATCH (a:Person) RETURN a.name AS name, a.age AS age",
+    database_="neo4j",
+)
+for person in records:  # <1>
+    print(person)
+    # person.name or person.age are also valid
+
+# Some summary information  <2>
+print(f"Query `{query}` returned {records_count} records in {time} ms.".format(
+    query=summary.query, records_count=len(records),
+    time=summary.result_available_after
+))
+
+print("Available keys are {keys}")  # ['name', 'age'] <3>
```

Comment:
```suggestion
print("Query `{query}` returned {records_count} records in {time} ms.".format(
    query=summary.query, records_count=len(records),
    time=summary.result_available_after
))

print(f"Available keys are {keys}")  # ['name', 'age'] <3>
```
I think. Didn't run the snippet.

Reply:
Definitely. Had it right when I ran it, got it wrong when I fiddled with it after pasting :man_facepalming: